pkg/pillar/containerd/containerd.go

```diff
@@ -112,32 +110,23 @@ func containerdLoadImageTar(filename string) (map[string]images.Image, error) {
 	return names, nil
 }
 
-// CtrRm remove an existing container. If silent is true, then operation failures are ignored and no error is returned
-func CtrRm(containerPath string, silent bool) error {
-	log.Infof("ctrRm %s\n", containerPath)
+// SnapshotRm removes existing snapshot. If silent is true, then operation failures are ignored and no error is returned
+func SnapshotRm(containerPath string, silent bool) error {
+	log.Infof("snapshotRm %s\n", containerPath)
 
 	containerID := filepath.Base(containerPath)
```

Comment:
Rename `containerID` to `snapshotID` and get rid of `getSnapshotName(containerID)`?

Reply:
Again, seconded.

Reply:
Fixed